syntax = "proto3";

message GetRoomInfoC2S {
    
}